src/pathfinder.cpp

```diff
@@ -855,7 +855,7 @@ namespace fasttrips {
         NamedWeights::const_iterator ivt_weight = weights.find(ivt_str);
         if ((fare_attr != attributes.end()) && (ivt_weight != weights.end())) {
             //       (60 min/hour)*(hours/vot currency)*(ivt_weight) x (currency)
-            cost += (60.0/path_spec.value_of_time_) * ivt_weight->second * fare_attr->second;
+            cost += (60.0/path_spec.value_of_time_) * ivt_weight->second.weight_ * fare_attr->second;
```

Comment:
When @lmz made the changes to support the different growth curves in C++, she changed NamedWeights from a simple dictionary to a more complex structure. I put in this change and the one below to get everything to compile. This needs and extra set of eyes from @lmz or @e-lo.

Reply:
I looked at this but I'm not sure; this bit of code was introduced by @bhargavasana related to the utils (https://github.com/BayAreaMetro/fast-trips/commit/4410852eb4d4ad2b7b1888e8637ead54aa930899)

Reply:
Hi Clint, the changes you made look good to me!